monitoring/daily_reporter/src/slack.py

```diff
@@ -29,7 +29,7 @@ def _get_slack_message(label, ingests):
         if ingests.get(status, []):
             result += "\n" + status.title() + ":"
             for i in ingests[status][:15]:
-                result += f"\n- <https://ingest-inspector.wellcomecollection.org/ingests/{i['id']}|`{i['id']}`> – {i['space']}/{i['externalIdentifier']}"
+                result += f"\n- <https://ingest-inspector.wellcomecollection.org/?ingestId={i['id']}|`{i['id']}`> – {i['space']}/{i['externalIdentifier']}"
```

Comment:
The URL that the daily reporter currently uses is that of the Glitch ingest inspector. Rather than the modified one here (which also does not resolve). I think the deployment for the daily reporter of the last mosst recent change was missed due to the lack of automation.